### PR TITLE
DX-2217 Updated App to Use Multiple Endpoints

### PR DESCRIPTION
**Pull request by ckoegel**

### CODE REVIEW

**Review by ajrice6713 — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -16,16 +16,19 @@
...
   "text":"Hello World!"
 }
 ```
-The application will text the number `+19994444` the words `Hello World!`.
+The application will text the number `+19199994444` the words `Hello World!`.
 If you text your Bandwidth number you will see your text printed out to the logs.
 
+The other two endpoints are used for handling inbound and outbound webhooks from Bandwidth. In order to use the correct endpoints, you must check the "Use multiple callback URLs" box on the application page in Dashboard. Then in Dashboard, set the INBOUND CALLBACK to `/callbacks/inbound/messaging` and the STATUS CALLBACK to `/callbacks/outbound/messaging/status`. The same can be accomplished via the Dashboard API by setting InboundCallbackUrl and OutboundCallbackUrl respectively.
+
+Inbound callbacks are sent notifying you of a received message on a Bandwidth number, this app prints the phone numbers invloved, as well as the text received. Outbound callbacks are status updates for messages sent from a Bandwidth number, this app has a dedicated response for each type of status update.
```
Comment (hamermike) [inline README.md]:
```suggestion
Inbound callbacks are sent notifying you of a received message on a Bandwidth number, this app prints the phone numbers involved, as well as the text received. Outbound callbacks are status updates for messages sent from a Bandwidth number, this app has a dedicated response for each type of status update.
```

**Review by hamermike — CHANGES_REQUESTED** [review]

**Comment by hamermike** [comment]
@ckoegel I'm assuming the controllers do not send a default success status code (ignore me if they do) so we should be sending an OK back to Bandwidth.

File: README.md [hunk 2]
```diff
@@ -16,16 +16,19 @@
 <!-- /TOC -->
 
 # Description
-Using a tool capable of posting (Postman) post a json body to the apps endpoint `/messages` with a json body like:
+Using a tool capable of making POST requests (Postman), send a POST request to the app's endpoint `/callbacks/outbound/messaging` with a json body like:
 ```json
 {
-  "to": "+19994444",
+  "to": "+19199994444",
```
Comment (abaldwin-Bandwidth) [inline README.md:22]: is that a Bandwidth number our account owns?

Comment (ckoegel) [reply README.md:22]: I don't believe so, I think its just a random number tbh

**Review by hamermike — APPROVED** [review]

**Review by ajrice6713 — APPROVED** [review]

**Pull request merged**